scene/gui/dialogs.cpp

```diff
@@ -90,7 +97,6 @@ void AcceptDialog::_notification(int p_what) {
 			}
 		} break;
 
-		case NOTIFICATION_READY:
```

Comment:
This removal doesn't seem relevant to the fix.

Reply:
You're right. I'm sorry, I used that notification in the first commit so I deleted the empty one. When I changed it to `NOTIFICATION_POST_TREE_ENTERED` I forgot to add it again. I'm updating it now!